CONTRIBUTING.md

```diff
@@ -137,18 +137,21 @@ If we accept the PR, a [maintainer](MAINTAINERS.md) will merge your change and u
 
 If we reject the PR, we will close the pull request with a comment explaining why. This decision isn't always final: if you feel we have misunderstood your intended change or otherwise think that we should reconsider then please continue the conversation with a comment on the PR and we'll do our best to address any further points you raise.
 
-## Feature Branches
+## Using Feature Branches
```

Comment:
Please update the TOC entry with the updated name for this section and add the new section name